src/sentry/tagstore/snuba/backend.py

```diff
@@ -586,13 +587,21 @@ def get_tag_value_paginator_for_projects(
 
         conditions = []
 
-        if snuba_key in BLACKLISTED_COLUMNS:
-            snuba_key = "tags[%s]" % (key,)
-
-        if query:
-            conditions.append([snuba_key, "LIKE", u"%{}%".format(query)])
+        if key in SearchVisitor.numeric_keys and snuba_key not in BLACKLISTED_COLUMNS:
```

Comment:
I'm not sure if we'd want this to apply to all numeric keys. Maybe that's the spec though.

I'm thinking for numeric keys like project.id and issue.id, and others that it doesn't make sense to span the condition to be -50, + 50 of the value. Maybe we can define a "fuzzy_numeric_keys" as a list of keys where this kind of search makes sense.

Reply:
I just realized project_id is in BLACKLISTED_COLUMNS. Does project.id get the same treatment? What about issue_id and issue.id?

Reply:
:+1: added a list of keys for this search type that excludes both issue/project id.

As far as I can tell this tag value function isn't called for project/issue ids since neither get treated as valid tags by our search bars